.travis.yml

```diff
@@ -22,6 +22,7 @@ before_install:
 
 before_script:
   - git diff --exit-code yarn.lock
+  - yarn check
```

Comment:
this is interesting, there are two different cases these commands solve for -

- yarn check - you committed an incoherent set of module requirements
- git diff - the yarn.lock file needs to be updated in this PR

we should check for both across all our repos.